spec/unit/coinbase/staking_operation_spec.rb

```diff
@@ -344,7 +344,7 @@
   end
 
   describe '#failed_state?' do
```

Comment:
Can clean this up in a follow-up, but these names should change as well

Reply:
Fixed 👍